features/user_has_a_profile_page.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Use has a profile page
+  As a user
+  In order to manage my account
+  I would like to have a user profile page
+
+Background:
+  Given We have the following user:
+      | email             |
+      | maran@test.com |
+      | author@test.com |
+
+  And We have the following recipes:
+      | title         | user           | forked from  |
+      | Pizza         | maran@test.com |              |
+      | Forked Pizza  | maran@test.com | Pizza        |
+
+  And I am logged in as "maran@test.com"
+
+Scenario: User sees his email, own recipes and forked recipes on his profile page
+  When I am on my profile page
```

Comment:
IMO a reveant part of the test (on this stage)  would be to check if you are able to navigate to the page by clicking on a link in the navbar.

Reply:
:+1: